stages/org.osbuild.firewall

```diff
@@ -92,6 +119,36 @@ def main(tree, options):
                        list(map(lambda x: f"--remove-service={x}", disabled_services)),
                        check=True)
 
+    for zone_item in zones:
+        # specifying an empty zone flag results in the source being applied to
+        # the default zone
+        zone_name = zone_item['name']
+        # check that the given zone exists, if not create it
+        if zone_name != "":
+            res = subprocess.check_output(["chroot",
+                                           tree,
+                                           "firewall-offline-cmd",
+                                           "--get-zones"])
+            if res.find(zone_name.encode()) < 0:
+                subprocess.run(["chroot",
+                                tree,
+                                "firewall-offline-cmd",
+                                "--permanent",
+                                f"--new-zone={zone_name}"],
+                               check=True)
+                subprocess.run(["chroot",
+                                tree,
+                                "firewall-offline-cmd",
+                                "--reload"],
+                               check=True)
```

Comment:
This is not documented in the man-pages, can you explain what this does? Also note that this runs on an unbooted system, so we really don't want to reload anything, do we?

Reply:
I've always seen a reload after a new zone was added, but yeah we haven't booted yet, so I will delete it :+1:

Reply:
Weird thing is, it is not listed in the man-page. But yeah, lets just drop it.